.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
         args: [--ignore-missing-imports]
         additional_dependencies: [types-PyMySQL==1.0.19, types-requests==2.27.30]
 -   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Seems there was an incompatibility with poetry, was raising errors. https://github.com/PyCQA/isort/releases/tag/5.11.5

 Bumped it to latest version.

Reply:
It seems we had the same reaction at the same time. Could you look at #139 which also updates the other checks?

Reply:
yeeeeh was going to say the same haha

Reply:
will revert my changes from here

Reply:
I may just merge this one to unblock main, and follow up with the other updates